app/src/main/java/de/danoeh/antennapod/config/StorageCallbacksImpl.java

```diff
@@ -125,6 +125,7 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                     PodDBAdapter.KEY_CHAPTER_TYPE));
         }
         if(oldVersion <= 14) {
```

Comment:
Do we need to increase the DB version?  Also, should probably be a constant, no?